eventbrite.py

```diff
@@ -0,0 +1,422 @@
+    # Create the event details JSON
+    event_detail = {
+        "event": {
+            "name": {
+                "html": f"<p>{title}</p>"
+            },
+            "description": {
+                "html": f"<p>{description}</p>"
+            },
+            "start": {
+                "timezone": "UTC",
+                "utc": "2023-11-12T02:00:00Z"
+            },
+            "end": {
+                "timezone": "UTC",
+                "utc": "2023-11-30T02:00:00Z"
+            },
+            "currency": "AUD",
+            "online_event": False,
+            "organizer_id": "",
+            "listed": False,
+            "shareable": False,
+            "invite_only": False,
+            "show_remaining": True,
+            "password": "12345",
+            "capacity": 100,
+            "is_reserved_seating": False,
+            "is_series": False,
+            "show_pick_a_seat": False,
+            "show_seatmap_thumbnail": False,
+            "show_colors_in_seatmap_thumbnail": False,
+            "locale": "en_AU"
+        }
+    }
+
+    # Convert the event details to a JSON string
+    event_detail_json = json.dumps(event_detail)
+
+    # Create the event
+    new_event = create_event(http,eventbrite_api_token, organization_id, event_detail_json)
+
+    # Extract the event_id from the response
+    event_id = new_event['id']
+
+    # Create the ticket class details JSON
+    ticket_detail = {
+        "ticket_class": 
+        {
+            "name": "API Test1",
+            "donation": "false",
+            "quantity_total": "100",
+            "minimum_quantity": "1",
+            "maximum_quantity": "10",
+            "hidden": "false",
+            "auto_hide": "false",
+            "auto_hide_before": "",
+            "auto_hide_after": "",
+            "sales_channels": ["online","atd"],
+            "hide_sale_dates": "false",
+            "delivery_methods": ["electronic"],
+            "include_fee": "false",
+            "sales_start": "2023-11-09T08:00:00Z",
+            "sales_end": "2023-11-11T03:00:00Z",
+            **({"cost": f"AUD,{cost_cents}", "free": "false"} if cost_cents is not None else {'free': "true"}),
+        }
+    }
+
+    # Convert the ticket class details to a JSON string
+    ticket_detail_json = json.dumps(ticket_detail)
+
+    # Create the ticket class
+    new_tickets = create_ticket_class(http, eventbrite_api_token, event_id, ticket_detail_json)
+    if publish:
+        publish_event(http,eventbrite_api_token, event_id)
+    return new_event, new_tickets
+
+if __name__ == "__main__":
+
+    # Retrieve the values of an environment variables
+    eventbrite_api_token = os.environ.get('EVENTBRITE_API_TOKEN')
+    organization_id = os.environ.get('EVENTBRITE_ORGANIZATION_ID')
+
+    # Create a PoolManager object
+    http = urllib3.PoolManager()
+
+    # quick_create_event(http, eventbrite_api_token, organization_id, "API-TEST-01", "API-TEST-DESCRIPTION", 2000, True)
```

Comment:
Don't comment this out, perhaps make publish default to False instead. You can even past a mandatory environment variable to control whether to publish or not.

Reply:
For the PUBLISH environment variable, that one you can use `.get`. Default can be either true or false.